Components/Text Utilities/text utils.css

```diff
@@ -0,0 +1,17 @@
+@import "../../Colors/colors.css";
+
+.heading {
+  font-size: 3rem;
+}
+
+.small-text {
```

Comment:
1.5rem seems a lot for small text you can decrease it to a more appropriate value.

Reply:
Will update in next PR. Thanks for the suggestions captain 🤝